src/main/resources/org/codehaus/mojo/servicedocgen/generation/velocity/OpenApi.json.vm

```diff
@@ -0,0 +1,267 @@
+{
+  "head": {
+    "title": "$services.info.title",
+    "meta": [
+      {
+        "charset": "utf-8"
+      }
+    ],
+    "link": [],
+    "script": []
+  },
+  "headers": [
+    {
+      "tagName": "h1",
+      "textContent": "$services.info.title",
+      "attributes": []
+    },
+    {
+      "tagName": "h2",
+      "textContent": "$service.name:\n            $service.description",
+      "attributes": {
+        "class": "ellipsis"
+      }
+    },
+    {
+      "tagName": "h3",
+      "textContent": "$operation.httpMethod\n                  $service.basePath$operation.path",
```

Comment:
I am not very experienced with OpenAPI but looking at this I can quickly see that this has not been tested really.
There are no velocity macros in here to loop over the services and for each service over the operations.
Sorry to say so but this can never have worked.
Get some hints here:
https://github.com/mojohaus/servicedocgen-maven-plugin/blob/master/src/main/resources/org/codehaus/mojo/servicedocgen/generation/velocity/Service-Documentation.html.vm#L493
https://github.com/mojohaus/servicedocgen-maven-plugin/blob/master/src/main/resources/org/codehaus/mojo/servicedocgen/generation/velocity/Service-Documentation.html.vm#L512

Also when writing velocity code, you should study the velocity documentation:
https://velocity.apache.org/engine/devel/user-guide.html